profiler/src/ProfilerEngine/Datadog.Profiler.Native/ManagedThreadInfo.cpp

```diff
@@ -20,7 +20,7 @@ std::uint32_t ManagedThreadInfo::GenerateProfilerThreadInfoId(void)
 }
 
 ManagedThreadInfo::ManagedThreadInfo(ThreadID clrThreadId) :
-    ManagedThreadInfo(clrThreadId, 0, static_cast<HANDLE>(0), shared::WSTRING())
+    ManagedThreadInfo(clrThreadId, 0, static_cast<HANDLE>(nullptr), shared::WSTRING())
```

Comment:
I'm not sure HANDLE should be seen as a pointer: most of the time it is an offset in a table so 0 makes more sense than nullptr

Reply:
handle are pointer according to the typdef def. 
But ok let's leave 0